src/icalendar/cal/calendar.py

```diff
@@ -85,6 +85,18 @@ def example(cls, name: str = "example") -> Calendar:
 
     @classmethod
     def from_ical(cls, st, multiple=False):
+        """Parse iCalendar data into Calendar instances.
+
+        Wraps :meth:`Component.from_ical() <icalendar.Component.from_ical>` with
```

Comment:
I think this should link to the Component module, instead of the icalendar alias. Compare:

- https://icalendar--1055.org.readthedocs.build/en/1055/reference/api/icalendar.html#icalendar.Component.from_ical
- https://icalendar--1055.org.readthedocs.build/en/1055/reference/api/icalendar.cal.component.html#icalendar.cal.component.Component.from_ical

```suggestion
        Wraps :meth:`Component.from_ical() <icalendar.cal.component.Component.from_ical>` with
```